SSDynamicText/NSAttributedString+SSTextSize.h

```diff
@@ -3,10 +3,10 @@
 //  SSDynamicText
 //
 //  Created by Remigiusz Herba on 28/08/15.
-//
+//  Copyright (c) 2015 Splinesoft. All rights reserved. 
 //
 
-#import <Foundation/Foundation.h>
+@import Foundation;
```

Comment:
Right now `SSDynamicLabel`, `SSDynamicTextView` and `SSDynamicTextField` have `@import` and everything works.
Could you write example when you had a problem with pod without `-fmodules` compiler flag?

Thanks in advance!